hashes/zkevm/src/keccak/tests.rs

```diff
@@ -126,12 +163,21 @@ fn verify<F: Field + Ord + FromUniformBytes<64>>(
     _success: bool,
 ) {
     let k = config.k;
-    let circuit = KeccakCircuit::new(config, Some(2usize.pow(k) - 109), inputs);
+    let circuit = KeccakCircuit::new(config, Some(2usize.pow(k) - 109), inputs, true);
 
     let prover = MockProver::<F>::run(k, &circuit, vec![]).unwrap();
     prover.assert_satisfied();
 }
 
+fn extract_value<'v, F: Field>(assigned_value: KeccakAssignedValue<'v, F>) -> F {
```

Comment:
The current one is pretty ad-hoc. I will polish it and plan to add it into `halo2-base` soon.

As for adding `From/Into`:
`AssignedCell` has a good design - it intentionally prevents people from reading witness values in syntax level. It highly reduces the probability that people construct a different circuit just with a different input.
Therefore this is actually an unsafe operation which only testing and halo2-lib might need it. I don't want to allow people easily do it so adding `From/Into` might be inappropriate.

As for adding into `halo2_proofs`:
Adding this into `halo2_proofs` might make more sense than adding into `halo2_base`... @jonathanpwang WDYT?

Reply:
I think it is questionable whether `AssignedCell` is actually good design, since you can always unwrap the `Value` if you need it, and it makes the developer experience rather clunky.

That being said, since it is the current design, adding `Into`/`From` does seem too opposite the design intent. We prefer not to make too many changes to the `halo2_proofs` backend itself, so they should just stay as some external tooling.